docs/examples/quick-start-with-terraform.md

```diff
@@ -0,0 +1,212 @@
+# Using Terraform to play with Liqo
+
+Terraform is a widely used Infrastructure as Code (IaC) tool that allows engineers to define their software infrastructure in code.\
+This tutorial aims at presenting how to set up an environment with Liqo installed via Terraform and explore the Liqo capabilities through its Provider and Resources.
```

Comment:
I would personally suggest to better emphasize what a reader is expected to learn reading this tutorial.